mne/minimum_norm/tests/test_inverse.py

```diff
@@ -204,33 +196,30 @@ def _compare_io(inv_op, out_file_ext='.fif'):
     _compare(inv_init, inv_op)
 
 
-@testing.requires_testing_data
```

Comment:
how does the fixture work? now every test requires the testing data?

Reply:
Nope, ones that use the new `evoked` and `cov` fixtures (currently just these tests in `minimum_norm` and `lcmv`) are automatically marked as requiring the test data by the fixtures themselves.